database/factories/CustomerFactory.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Customer>
+ */
+class CustomerFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        // config/app.phpで'faker_locale'を変更した場合は日本語対応している項目は日本語で生成される
+        $faker = \Faker\Factory::create('ja_JP');
+
+        return [
+            'name' => $faker->name(),
+            'mail' => $faker->email(),
+            'zipcode' => $faker->postcode(),
+            'address' => $faker->address(),
```

Comment:
これだと郵便番号もaddressに含まれてしまいます。
先頭の8文字が郵便番号(7文字+空白1文字)になっているので、9文字目以降をaddressに設定します。
```suggestion
            'address' => mb_substr($faker->address(), 9),
```